### PR TITLE
Added Build Instructions in README.md

### PR DESCRIPTION
**Pull request by alfinwilliam**

### CODE REVIEW

**Comment by tachyons** [comment]
Thanks, this expects build environment to be Linux, we support windows and Mac environment as well. Can you add them or specify given instructions are for Linux ?

**Comment by alfinwilliam** [comment]
Sorry. It was specifically for Linux. I'm not much familiar with MacOS & Windows. Should I edit it out and add the Linux tag?

**Comment by alfinwilliam** [comment]
I refereed some other qt repositories and revamped the instructions. Kindly look into it.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,24 @@ Olam free and open source English Malayalam repository with Qt gui
...
 * Malayalam Malayalam
 
+# Building from source
+
+### General Build Requirements
+* Qt 5.9 SDK or later
+
+### Build Setup Steps
+1. Install the latest Qt SDK (and optionally, the Qt Creator IDE) from https://www.qt.io/download
+    * You can install Qt via Homebrew on macOS, but you will need to use `brew install qt --with-debug` to be able to create debug builds of Moonlight.
```
Comment (tachyons) [inline README.md]: Moonlight is a name of another app written in at ;-)

File: README.md [hunk 2]
```diff
@@ -12,6 +12,24 @@ Olam free and open source English Malayalam repository with Qt gui
 * Malayalam English
 * Malayalam Malayalam
 
+# Building from source
+
+### General Build Requirements
+* Qt 5.9 SDK or later
```
Comment (tachyons) [inline README.md]: I haven't actually tested with different versions of Qt, but ideally At 5.1 + should be enough

File: README.md [hunk 3]
```diff
@@ -12,6 +12,24 @@ Olam free and open source English Malayalam repository with Qt gui
...
+
+### Build Setup Steps
+1. Install the latest Qt SDK (and optionally, the Qt Creator IDE) from https://www.qt.io/download
+    * You can install Qt via Homebrew on macOS, but you will need to use `brew install qt --with-debug` to be able to create debug builds of Moonlight.
+    * You may also use your Linux distro's package manager for the Qt SDK as long as the packages are Qt 5.9 or later.
+    * This step is not required for building on Steam Link, because the Steam Link SDK includes Qt 5.9.
```
Comment (tachyons) [inline README.md]: We do not support stream link as it is a gaming platform

File: README.md [hunk 4]
```diff
@@ -12,6 +12,24 @@ Olam free and open source English Malayalam repository with Qt gui
...
+2. Run `git submodule update --init --recursive` from within `olam/`
```
Comment (tachyons) [inline README.md]: We don't have any git submodules for this project at the moment

**Review by tachyons — CHANGES_REQUESTED** [review]
Your initial version was better, just specify Linux there, new version seems copied from moonlight and they have custom setup

**Comment by alfinwilliam** [comment]
I reverted the commit and added the ubuntu tag. Sorry for all the trouble.

**Review by tachyons — APPROVED** [review]

**Comment by tachyons** [comment]
Thanks

**Pull request merged**